packages/graphql-auth-transformer/src/ModelAuthTransformer.ts

```diff
@@ -469,7 +586,9 @@ Static group authorization should perform as expected.`
 
     private validateRules(rules: AuthRule[]) {
         for (const rule of rules) {
-            const { queries, mutations, operations } = rule;
+            const { queries, mutations, operations,
```

Comment:
Missing validations:
- groupField and groupClaim is mutually exclusive
- identityField and identityClaim is mutually exclusive